### ISSUE TITLE
Something wrong with the magnetizability code?

### ISSUE THREAD

**Issue by susilehtola** [pyscf/pyscf#606]
For some reason I'm getting wrong magnetizabilities on the master branch. In contrast, my Fedora packaged version 1.7.1 yields values that agree with other programs.

Given the input
```
from pyscf import gto
from pyscf import scf
from pyscf import dft
from pyscf.prop.magnetizability.rks import Magnetizability
import numpy

mol=gto.M(
  verbose=0,
  atom='''
 h
 f 1 0.90
''',
  basis = 'aug-cc-pVDZ')

mf = dft.RKS(mol)
mf.xc = 'pbe'
mf.grids.atom_grid = (100, 590)
mf.conv_tol_grad = 1e-7
mf.run()
mag = Magnetizability(mf).kernel()

print('Tr chi = {:.10f}'.format(numpy.trace(mag)))
```
the output should be
```
Tr chi = -6.8808732594
```
but what I get from master is
```
Tr chi = -7.0548003731
```

**Comment by susilehtola** [comment]
Hm, if I build 1.7.1 from source I also get the wrong result.

But, if I disable building libcint, using the Fedora packaged version, I get the right result even in master!

**Comment by susilehtola** [comment]
This is quite odd given that I've built both versions of libcint with OpenBLAS, using `libopenblaso` which is the thread-safe, OpenMP version of the library.

**Comment by susilehtola** [comment]
Actually, this is even weirder: I actually had `qcint` installed, not `libcint`. The former gives the right result, the latter the wrong one, even though both are compiled in the same way!

If you modify `pyscf/lib/CMakeLists.txt` as
```
-    GIT_REPOSITORY https://github.com/sunqm/libcint.git
+    GIT_REPOSITORY https://github.com/sunqm/qcint.git
```
you should be able to reproduce the discrepancy.

**Comment by sunqm** [comment]
Thanks for tracking down the problem. It's caused by a bug in libcint ggkin. It was fixed in commit b2824f0d21663dbeabd103388dbfaff090a6394d

**Comment by susilehtola** [comment]
Thanks, looks like updating to libcint 3.0.20 indeed fixes the problem.
